mantidproject/mantid
# Handle spectrum properties when loading processed nexus

Description of work.
This PR removes commented out code out of `LoadNexus`. Previously `LoadNexusProcessed` could not handle a spectrumList and spectrumMin/Max so a check was in place to prevent that. 
However since then the ability to handle these properties has been added so the check needs to be removed. Additionally some additional simple validation was added. 

**To test:**
Open a processed nexus file. A test file called 'GEM3870_focussed_legacy' within the unit test files can be used. Try to enter some invalid input such as reversing Min/Max or setting them to negative numbers - it should not allow this.
Fixes #15958 .

Release notes do not need to be updated

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
